### DISCUSSION TITLE
Speaker.pth generating instead of Speaker.json

### DISCUSSION THREAD

**Discussion by SyGen899** [coqui-ai/TTS#1688] [General Q&A]
Hey, newbie here, not sure if Im installing the package correctly or doing something wrong, but Ive tried to train a VITS model locally but struggled to learn anything because of low resources, so I tried on colab, but my model is storing a speaker.pth file and not a speaker.json like my local run, is this correct? or am I messing something up here.

**Comment by p0p4k** [comment]
Hello, to help you with debugging your code, we need to see your code first. Please post your train.py and train log so that we can help you further. Thanks.

**Reply by mbarnig** [reply]
I recently trained the `tts_models--multilingual--multi-dataset--your_tts` with version 0.7.1 and I noticed that the informations about speakers are now saved in in a speaker.pth file.

This file contains the same content as the former speaker.json file, in a binary format, for example :    

`.... q\00(X\00\00\00EdqK\00X\00\00\00ElliotqKX\00\00\00EzwaqKX\00\00\00Gillesq....` 

In my case the speaker names Ed, Elliot, Ezwa, Gilles , etc are included, probably with the related indices.

So you didn't mess something up, this is the new correct behavior.

**Comment by erogol** [comment]
we switch json to pth for faster IO. But old son files should also work
